examples/plt_cfg.yaml

```diff
@@ -0,0 +1,85 @@
+plot_dir: /home/output_dir
+timestamp: False
+results_default:
+  atlas_second_tag: $\sqrt{s} = 13.6$ TeV, MC23
+  
+taggers_config: taggers.yaml
+taggers: 
+  - dummy1
+  - dummy2
+  - dummy3
+reference_tagger: dummy1
+
+
+sample:
+  name: ttbar
+  cuts:
+    - "pt > 20000"
+    - "pt < 250000"
+    - "eta < 2.5"
+    - "eta > -2.5"
+  str: $t\overline{t}$ 20 < $p_T$ < 250 GeV, $|\eta| < 2.5$
+
+roc_plots:
+  -  args:
+      signal: cjets
+      x_range: [0.5, 1.0]
+      plot_kwargs:
+        figsize: [8.5, 7.5]
```

Comment:
Would be nice to maybe have a default here so this doesn't have to be specified

Reply:
So both the x_range and the figsize include default values (0.5, 1.0) for the x-range and whatever puma default is for figsize. I include them here in the example to make it clear how things can be modified.

Reply:
cool! thanks